beacon-chain/rpc/eth/blob/handlers.go

```diff
@@ -131,6 +134,41 @@ func (s *Server) Blobs(w http.ResponseWriter, r *http.Request) {
 		network.WriteError(w, errJson)
 		return
 	}
+	ssz, err := beacon_middleware.SszRequested(r)
+	if err != nil {
+		errJson := &network.DefaultErrorJson{
+			Message: err.Error(),
+			Code:    http.StatusInternalServerError,
+		}
+		network.WriteError(w, errJson)
+		return
+	}
+	if ssz {
+		v2sidecars, err := migration.V1Alpha1BlobSidecarsToV2(sidecars)
+		if err != nil {
+			errJson := &network.DefaultErrorJson{
+				Message: err.Error(),
+				Code:    http.StatusInternalServerError,
+			}
+			network.WriteError(w, errJson)
+			return
+		}
+		sidecarResp := &ethpb.SidecarsResponse{
```

Comment:
You need to define `SidecarsResponse`. You can use `BlobSidecars` because SSZ does not preserve field names. It only stores offsets and binary data.